ml-agents/mlagents/model_serialization.py

```diff
@@ -60,27 +60,40 @@
 class SerializationSettings(NamedTuple):
     model_path: str
     brain_name: str
+    checkpoint_path: str = ""
```

Comment:
Is this field just used for changing the name of the exported .nn file?

Reply:
I guess the original intention was to have everything here be importable from the YAML file. So if this is being mutated it kind of defeats the purpose. 

I'd almost prefer a parameter to the export policy model that just does a filename override, then you'd also not need an `is_checkpoint` flag.

Reply:
Added the explicit export filepath to `export_policy_model`

Reply:
I was actually wrong about this one 🤦. Thought it was one of the `attrs` classes, this seems fine to me even with the checkpoint_path.